scripts/index.js

```diff
@@ -15,9 +14,7 @@ humberger.addEventListener('click', showMenu);
 menuClose.addEventListener('click', hideMenu);
 menuItemsNode.forEach((menuItem) => menuItem.addEventListener('click', hideMenu));
 
-/* global testimonials, parseVirtualDom, TestimonialVirtualDom */
-testimonials.forEach((testimonial, index) => (
-  testimonialContainer.appendChild(
-    parseVirtualDom(TestimonialVirtualDom({ ...testimonial, index })),
-  )
-));
\ No newline at end of file
+// /* global projects, parseVirtualDom, WorkVirtualDom */
+// projects.forEach((project, index) => (
+//   worksSection.appendChild(parseVirtualDom(WorkVirtualDom({ ...project, index })))
+// ));
```

Comment:
[Optional] Kindly remove commented pieces of code in order to clean up the code base.